pkg/k8s/service_cache.go

```diff
@@ -230,22 +230,18 @@ func (s *ServiceCache) deleteEndpoints(svcID ServiceID, swg *lock.StoppableWaitG
 
 	svc, serviceOK := s.services[svcID]
 	delete(s.endpoints, svcID)
-	endpoints, serviceReady := s.correlateEndpoints(svcID)
+	endpoints, _ := s.correlateEndpoints(svcID)
 
 	if serviceOK {
 		swg.Add()
 		event := ServiceEvent{
-			Action:    DeleteService,
+			Action:    UpdateService,
```

Comment:
What will happen if the user deletes the service first and then the endpoints? Will it show up in the datapath?

Reply:
The second event will be noop, so no endpoints in the datapath BPF maps.